string name
string camera_name
geometry_msgs/PoseStamped pose